contracts/src/MultiTroveGetter.sol

```diff
@@ -126,7 +126,7 @@ contract MultiTroveGetter is IMultiTroveGetter {
         assert(address(sortedTroves) != address(0));
 
         data = new DebtPerInterestRate[](_maxIterations);
-        currId = sortedTroves.getPrev(_startId);
+        currId = _startId == 0 ? sortedTroves.getPrev(_startId) : _startId;
```

Comment:
Good catch! We could also use `sortedTroves.getLast()` when `_startId == 0`, which might be more readable.